### ISSUE TITLE
map_table(multipoints) object table is wrong

### ISSUE THREAD

**Issue by mdsumner** [mdsumner/spbabel#29]

**Comment by mdsumner** [comment]
multipoints fixed in 82f0c46b6cf7145d3e9bacc30c11031da58d624e  but single-points needs to either special-case shortcut to object_ id on vertices, or put in dummy branches.

**Comment by mdsumner** [comment]
Fixed. 

```R
pts <- sp(holey %>% select(x_, y_, object_) %>% mutate(object_ = row_number()))
# plot(pts)
 map_table(pts)
```

```
$o
# A tibble: 50 × 2
   rownumber_    object_
        <int>      <chr>
1           1 7OKGYV2o3D
2           2 hsqPnWBSys
3           3 eZeUwKPvV4
4           4 RgRLbIQZMm
5           5 Zp6kKaHlqI
6           6 zN38c26rLc
7           7 igMThnfbZM
8           8 itUGJLegsb
9           9 03zDhDqDPq
10         10 8pu8FWVxMN
# ... with 40 more rows

$b
# A tibble: 50 × 2
      object_    branch_
        <chr>      <chr>
1  7OKGYV2o3D uURwNYBGaF
2  hsqPnWBSys m6r3A7uwww
3  eZeUwKPvV4 fcVRiQK3gi
4  RgRLbIQZMm IEIBIVMwTL
5  Zp6kKaHlqI 6eAf57LsWe
6  zN38c26rLc uElExJvfpf
7  igMThnfbZM 1qvwudXag1
8  itUGJLegsb f1fNaIhROS
9  03zDhDqDPq 4OP5znLa4X
10 8pu8FWVxMN SOm3o5YKvf
# ... with 40 more rows

$bXv
# A tibble: 50 × 2
      branch_    vertex_
        <chr>      <chr>
1  uURwNYBGaF hXxYKadFpU
2  m6r3A7uwww 7NAL4DUgsw
3  fcVRiQK3gi 8lnURQnSaj
4  IEIBIVMwTL KUSmgTbV5L
5  6eAf57LsWe hXxYKadFpU
6  uElExJvfpf gW8rslor4p
7  1qvwudXag1 bsCLJkKNJ2
8  f1fNaIhROS TQXInlEXJc
9  4OP5znLa4X QpHmljkE3b
10 SOm3o5YKvf gW8rslor4p
# ... with 40 more rows

$v
# A tibble: 27 × 3
      x_    y_    vertex_
   <dbl> <dbl>      <chr>
1      0     0 hXxYKadFpU
2      0    19 7NAL4DUgsw
3     46    19 8lnURQnSaj
4     46     0 KUSmgTbV5L
5      7     6 gW8rslor4p
6     13     6 bsCLJkKNJ2
7     13    13 TQXInlEXJc
8      7    13 QpHmljkE3b
9     18     1 V5OXxuBRom
10    24     1 SB4tDmLdZZ
# ... with 17 more rows

attr(,"class")
[1] "map_table" "list"  
```
